Application/Frontend/components/FriendsColumn/AddFriendModal.tsx

```diff
@@ -1,88 +1,77 @@
+import { useState, useEffect } from 'react';
 import { useDisclosure } from '@mantine/hooks';
-import { Modal, Tooltip, ActionIcon, Text, Stack, Button, TextInput } from '@mantine/core';
-import { IconPlus } from '@tabler/icons-react';
-import { useState } from 'react';
-import { NewFriendModalProps } from '@/accordTypes';
-import { useEffect } from 'react';
+import { Modal, Text, Stack, Button, TextInput } from '@mantine/core';
+import { NewFriendModalProps } from '@/accordTypes'; // Make sure this path is correct for your project
+import { showNotification } from '@mantine/notifications';
```

Comment:
Great use of the built-in notification system from Mantine. Adding responsive notifications throughout the application is one of the next steps toward finalizing our application UX.